RobotArm/examples/example-normalized.json

```diff
@@ -1,35 +1,60 @@
 {
-  "id": "Robot.FlexEdgeRobot",
-  "type": "RobotArm",
-  "robotId": {
-    "type": "Text",
-    "value": "FlexEdgeRobot"
-  },
-  "robotModel": {
-    "type": "Text",
-    "value": "KR 12345b"
-  },
-  "toolId": {
-    "type": "Integer",
-    "value": 1
-  },
-  "jobCurrentState": {
-    "type": "Text",
-    "value": "Processing"
-  },
-  "axis": {
-    "type": "Array",
-    "value": [
-      30.0,
-      14.0,
-      -55.0,
-      174.0,
-      145.0,
-      -37.0
-    ]
-  },
-  "status": {
-    "type": "Text",
-    "value": "#P_ACTIVE"
-  }
-}
\ No newline at end of file
+  "id": "urn:ngsi-ld:Robot:876543",
```

Comment:
This is not consistent with the schema.json. Please fix accordingly

Reply:
We updated also the shema.json file, but it is in another pull request. Maybe we should work and agree on that one first.